src/lib/provable/bigint.ts

```diff
@@ -763,7 +764,7 @@ function rangeCheck(x: Field, bits: number, signed?: boolean) {
       Gadgets.rangeCheck32(x);
       break;
     case 48:
-      rangeCheck48(x);
+      Gadgets.rangeCheckN(48, x);
       break;
     case 64:
       Gadgets.rangeCheck64(x);
```

Comment:
If we are commiting to `rangeCheckN`, then I think the rest of them should be changed too.  This should not block this PR however, and should be documented in a new issue and prioritized accordingly.